plio/static/plio/docs/download_csv_README.md

```diff
@@ -44,9 +44,9 @@ This section will clarify what each of the `.csv` files in the folder contains:
   - `question_type`: the type of question (e.g. `mcq`)
   - `question_text`: the text of the question itself
   - `question_options`: the options for the question
-  - `question_correct_answer`: the correct answer for the given question. For `question_type = mcq`, this represents the index of the correct answer among the `question_options`. Here, `answer = 1` indicates that the first option has been marked as the correct answer.
+  - `question_correct_answer`: the correct answer for the given question. For `question_type = mcq`, this represents the index of the correct answer among the `question_options`. In this case, `question_correct_answer = 1` indicates that the first option has been marked as the correct answer.
 
-    For `question_type = checkbox`, it contains a list of indices corresponding to the options which are correct. Here, `answer = [2,3]` indicates that the second and third options have been marked as the correct answers.
+    For `question_type = checkbox`, it contains a list of indices corresponding to the options which are correct. In this case, `question_correct_answer = [2,3]` indicates that the second and third options have been marked as the correct answers.
```

Comment:
This case is not present in the example below no?
Also in the example below, I noticed a typo. MCQ is written as MCC